mapping.csv

```diff
@@ -366,6 +378,7 @@
 "^((b106-)?(swisyn|arhost)|roopirs|ashwho)$",swisyn,used by microsoft,https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Worm:Win32/Roopirs.A,https://www.solvusoft.com/en/malware/trojans/tr-swisyn-ubp-13/
 "^swizzor$",swizzor,used by microsoft
 "^(b106-)?swrort$",swrort,used by microsoft
+"^syndicasec$",syndicasec,used by microsoft
```

Comment:
malpedia maps this to wmighost: https://malpedia.caad.fkie.fraunhofer.de/details/win.wmighost

Reply:
I just noticed that this mapping is already in malpedia.csv so I remove it from this file